src/host/os_isdir.c

```diff
@@ -9,15 +9,15 @@
 #include "premake.h"
 
 #ifdef _WIN32
-#include <Windows.h>
+#include <windows.h>
 #endif
 
 int os_isdir(lua_State* L)
 {
 	struct stat buf;
 	const char* path = luaL_checkstring(L, 1);
 #ifdef _WIN32
-	int attr;
+	DWORD attr;
```

Comment:
Was this causing build issues? If not, I'd consider it out of scope of this PR.

Reply:
Compiler warning:

```
../../src/host/os_isdir.c: In function 'os_isdir':
../../src/host/os_isdir.c:37:50: warning: comparison of integer expressions of different signedness: 'int' and 'long unsigned int' [-Wsign-compare]
   37 |  else if ((attr = GetFileAttributesW(wide_path)) != INVALID_FILE_ATTRIBUTES)
      |    
```